src/lighteval/models/transformers/adapter_model.py

```diff
@@ -41,6 +41,10 @@
 
 @dataclass
 class AdapterModelConfig(BaseModelConfig):
+    """
+    This class is used to manage the configuration of adapter models. Adapter models are designed to extend or adapt a 
+    base model's functionality for specific tasks while keeping most of the base model's parameters frozen.
```

Comment:
You need to document the attributes too :)

Reply:
Updated as discussed.